Maple2.Server.Game/Model/Field/Actor/FieldPlayer.cs

```diff
@@ -236,6 +236,15 @@ bool UpdateStateSyncTimeTracking() {
                     Session.ConditionUpdate(ConditionType.riding, codeLong: Value.Character.MapId);
                 }
                 break;
+            case ActorState.EmotionIdle:
+                if (UpdateStateSyncTimeTracking()) {
+                    Field.SkillMetadata.TryGet(stateSync.Flag1EmotionId, 1, out var emote);
+                    if (emote == null) {
+                        break;
+                    }
+                    Session.ConditionUpdate(ConditionType.emotiontime, codeString: emote.Property.Emotion);
```

Comment:
You'll also need to implement target. It looks like for `emotiontime` this is the player's Map ID